site/en/docs/extensions/mv3/intro/mv3-migration/index.md

```diff
@@ -279,48 +265,42 @@ With Manifest V3 the `executeScript()` method also moves to a different API.
 If you use executeScript() anywhere in your code, you'll need to update that call to use the new
 API. The `insertCSS()` and `removeCSS()` methods similarly move from chrome.tabs to
 chrome.scripting.
-{% endAside %}
 
+{% endAside %}
 
 ### Remotely hosted code  {: #remotely-hosted-code }
 
-_Remotely hosted code_ refers to any code that is not included in an
-extension's package as a loadable resource. For example, both of the following
-are considered remotely hosted code:
+_Remotely hosted code_ refers to any code that is not included in an extension's package as a
+loadable resource. For example, both of the following are considered remotely hosted code:
 
 *   JavaScript files pulled from a remote server
 *   a code string passed into eval at runtime
 
-In MV3, all of your extension's logic must be bundled with the extension. You
-can no longer load and execute a remotely hosted file. A number of alternative
-approaches are available, depending on your use case and the reason for remote
-hosting. Two such approaches are:
-
-**Configuration-driven features and logic**—In this approach, your extension
-loads a remote configuration (for example a JSON file) at runtime and caches
-the configuration locally. The extension then uses this cached configuration to
-decide which features to enable.
+In MV3, all of your extension's logic must be bundled with the extension. You can no longer load and
+execute a remotely hosted file. A number of alternative approaches are available, depending on your
+use case and the reason for remote hosting. Two such approaches are:
 
-**Externalize logic with a remote service**—Consider migrating application
-logic from the extension to a remote web service that your extension can call.
-(Essentially a form of message passing.) This provides you the ability to keep
-code private and change the code on demand while avoiding the extra overhead of
-resubmitting to the Chrome Web Store.
+**Configuration-driven features and logic**—In this approach, your extension loads a remote
+configuration (for example a JSON file) at runtime and caches the configuration locally. The
+extension then uses this cached configuration to decide which features to enable.
 
+**Externalize logic with a remote service**—Consider migrating application logic from the extension
+to a remote web service that your extension can call. (Essentially a form of message passing.) This
+provides you the ability to keep code private and change the code on demand while avoiding the extra
+overhead of resubmitting to the Chrome Web Store.
 
 ### Executing arbitrary strings  {: #executing-arbitrary-strings }
 
 The `code` property from executeScript's
```

Comment:
I think you need to rewrite this line.

It should say something like "...previously, you used the `chrome.tabs` API to execute script in a parameter called `code`, but this has been replaced with calls in the `chrome.scripting` API which takes source files or function references.